### PR TITLE
DAOS-8935: enable pci event module to run in unprivileged process

### PR DESCRIPTION
**Pull request by tanabarr**
Add patch https://github.com/spdk/spdk/commit/148a9ab0c06346f9fec109a1df00651c1f5a0499
to be used by DAOS so that the daos_engine process can use the pci_event module when run
by an unprivileged/non-root user.

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+ --  Tom Nabarro <tom.nabarro@intel.com>  Tue, 09 Nov 2021 00:00:00 +0000
+
+spdk (21.07-2) unstable; urgency=medium
+  [ Tom Nabarro ]
   * Upgrade SPDK to 21.07 release + patch to enable multiple dpdk cli
-   opts on init.
+    opts on init.
 
- --  John Malmberg <John.e.malmberg@intel.com>  Fri, 13 Aug 2021 19:00:00 +0000
+ --  Tom Nabarro <tom.nabarro@intel.com>  Fri, 10 Aug 2021 00:00:00 +0000
```
Comment (brianjmurrell) [inline debian/changelog:13]: This seems an odd change.  This whole log of changes in fact looks quite different than what is in the RPM.

Can we update this file to match what is in the `%changelog`?

**Review by brianjmurrell — APPROVED** [review]
Looks good, but the `debian/changelog` looks out of sorts.  In particular the changing of attribution of one of the entries is troubling.  It would be nice to get that file in sync with the `%changelog`.

File: debian/changelog (hunk 1 above)
Comment (tanabarr) [reply debian/changelog:13]: yes it's out of sync, I will update (I updated the entry to match the comment as it was mismatched with the date/author)

**Comment by brianjmurrell** [comment]
> sync debian changelog with rpm spec %changelog

Appreciate that @tanabarr!

**Review by JohnMalmberg — APPROVED** [review]

**Comment by brianjmurrell** [comment]
You need to update this PR's `packaging/` with daos-stack/packaging#60 to work around the group repo issue with Nexus.

**Comment by brianjmurrell** [comment]
@tanabarr Please add @daos-stack/release-engineering to the reviewers when you are ready for this to land.

**Pull request merged**